skipper.go

```diff
@@ -1223,6 +1230,13 @@ func listenAndServeQuit(
 		}
 	}
 
+	log.Infof("Listen on %v", address)
+
+	l, err := listen(o, address, mtr)
+	if err != nil {
+		return err
+	}
```

Comment:
Now here is the single place where main listener is created (ignoring insecure listener) which opens up a way to wrap it e.g. to track connections.